### PR TITLE
Add types for follow-redirects

### PR DESCRIPTION
**Pull request by forivall**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/follow-redirects/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
+// Type definitions for follow-redirects 1.5
+// Project: https://github.com/olalonde/follow-redirects
+// Definitions by: Emily Klassen <https://github.com/forivall>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+/// <reference types="node" />
```
Comment (PranavSenthilnathan) [inline types/follow-redirects/index.d.ts]: Are there any types that you're using from node? I think this can be removed.

**Review by PranavSenthilnathan — CHANGES_REQUESTED** [review]

File: types/follow-redirects/index.d.ts (hunk 1 above)
Comment (forivall) [reply types/follow-redirects/index.d.ts]: http, https and stream require node to be included: 

When running lint:
```
ERROR: 7:27  expect  TypeScript@next compile error: 
Cannot find module 'http'.
ERROR: 8:28  expect  TypeScript@next compile error: 
Cannot find module 'https'.
ERROR: 9:26  expect  TypeScript@next compile error: 
Cannot find module 'stream'.
```

**Comment by forivall** [comment]
@PranavSenthilnathan I left a comment on the requested change. Let me know if it resolves the issue.

**Comment by BendingBender** [comment]
@forivall just open a new PR, just like typescript-bot says

**Pull request closed without merging**